doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+
++--------------+---------------------------------------------------------------+
+| Plutus Script Utils                                                          |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-script-utils``                           |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api                                |
++--------------+---------------------------------------------------------------+
+| State        | Work in progress with an unstable interface.                  |
++--------------+---------------------------------------------------------------+
+
+2. Marconi
+--------------------------
+
+`Marconi <https://github.com/input-output-hk/plutus-apps/tree/main/marconi>`_ is an 
+application for indexing useful data from the Cardano blockchain in a scalable way. 
+
++--------------+---------------------------------------------------------------+
+| Marconi                                                                      |
++==============+===============================================================+
+| Location     | ``plutus-apps/marconi``                                       |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api                                |
++--------------+---------------------------------------------------------------+
+| State        | In a very early alpha version. Has only a few indexers and    |
+|              | the final architecture is currently being designed.           |
++--------------+---------------------------------------------------------------+
+
+3. Plutus ledger
+--------------------------
+
+`Plutus ledger <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-ledger>`_ 
+is a "kitchen sink" set of components containing data types and functions that 
+complement `cardano-ledger <https://github.com/input-output-hk/cardano-ledger>`_ 
+related to Plutus. 
+
++--------------+---------------------------------------------------------------+
+| Plutus ledger                                                                |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-ledger``                                 |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, cardano-ledger, plutus-ledger-api,               |
+|              | plutus-script-utils                                           |
++--------------+---------------------------------------------------------------+
+| State        | Unstable interface with an uncertain future.                  |
++--------------+---------------------------------------------------------------+
+
+4. Plutus ledger constraints
+---------------------------------
+
```

Comment:
I'm confused about the difference between ledger constraints and Tx constraints. They seem to have the same description.

Reply:
If I'm understanding correctly, "Plutus Tx constraints" will eventually replace "Plutus ledger constraints." If so, let's move "Plutus Tx constraints" higher in the list to put more emphasis on it.